docs/DISTROS.md

```diff
@@ -0,0 +1,170 @@
+# curl distros
+
+<!-- markdown-link-check-disable -->
+
+Lots of organizations distribute curl packages to end users. This is a
+collection of pointers to where to learn more about curl on and with each
+distro.
+
+We discuss curl distro issues, patches and collaboration on the [curl-distros
+mailing list](https://lists.haxx.se/mailman/listinfo/curl-distros).
+
+# Alpine Linux
```

Comment:
The heading level is inconsistent. Some (e.g. Alpine, Clear, Conary) are level 1 whereas others (e.g. Arch, curl-for-win) are level 2.